src/com/lilithsthrone/game/character/PlayerCharacter.java

```diff
@@ -1649,6 +1651,14 @@ public boolean isDiscoveredWorldMap() {
 		return this.isQuestProgressGreaterThan(QuestLine.MAIN, Quest.MAIN_2_D_MEETING_A_LILIN);
 	}
 
+	public boolean isInActive() {
+		return !isActive;
+	}
```

Comment:
This is convoluted. if the `isActive` property is public, there is no reason to have this little condition bloating the code. Let the check be done when it has to be done.